R/catch-routine-registration.R

```diff
@@ -0,0 +1,6 @@
+# This dummy function definition is included with the package to ensure that
+# 'tools::package_native_routine_registration_skeleton()' generates the required
```

Comment:
It's slightly easier to do `tools::package_native_routine_registration_skeleton(".", character_only = FALSE)` which is what I did

Reply:
Hmmmm, overall I'm not sure automating this is the right approach because there is considerable variety in how people do symbol registration.  How does Rcpp do it?

Reply:
`Rcpp::compileAttributes()` effectively calls `tools::package_native_routine_registration_skeleton()` and then runs the regular code that generates RcppExports.

Adding this function definition ensures that this works transparently for existing workflows.

Reply:
Ah ok, perfect.